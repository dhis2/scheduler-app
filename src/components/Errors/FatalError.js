import React from 'react'
import { AbsoluteCenter } from '../AbsoluteCenter'
import InlineError from './InlineError'

const FatalError = props => (
    <AbsoluteCenter>
        <InlineError {...props} />
    </AbsoluteCenter>
)

export default FatalError
