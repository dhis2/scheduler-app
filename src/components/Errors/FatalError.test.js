import React from 'react'
import { shallow } from 'enzyme'
import FatalError from './FatalError'

describe('<FatalError>', () => {
    it('renders correctly', () => {
        const wrapper = shallow(
            <FatalError message="Error" details={['Details']} />
        )

        expect(wrapper).toMatchSnapshot()
    })
})
