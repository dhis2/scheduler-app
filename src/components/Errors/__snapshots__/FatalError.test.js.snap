// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FatalError> renders correctly 1`] = `
<AbsoluteCenter
  vertical={false}
>
  <InlineError
    details={
      Array [
        "Details",
      ]
    }
    message="Error"
  />
</AbsoluteCenter>
`;
